docs/xls-47d-price-oracles/ledger-object-types/oracle.md

```diff
@@ -0,0 +1,92 @@
+  "Provider": "70726F7669646572",
+  "AssetClass": "63757272656E6379",
+  "PriceDataSeries": [
+    {
+      "PriceData": {
+        "BaseAsset": "XRP",
+        "QuoteAsset": "USD",
+        "AssetPrice": 74,
+        "Scale": 2,
+      }
+    },
+  ],
+  "LastUpdateTime": 743609414,
+  "PreviousTxnID": "C53ECF838647FA5A4C780377025FEC7999AB4182590510CA461444B207AB74A9",
+  "PreviousTxnLgrSeq": 56865244
+}
+```
+
+
+## Oracle Fields
+
+| Field               | JSON Type | Internal Type | Required? | Description |
+|---------------------|-----------|---------------|-----------|-------------|
+| `Owner`             | String    | AccountID     | Yes       | The XRPL account with update and delete privileges for the oracle. It's recommended to set up [multi-signing](https://xrpl.org/set-up-multi-signing.html) on this account. |
+| `Provider`          | String    | Blob          | Yes       | An arbitrary value that identifies an oracle provider, such as Chainlink, Band, or DIA. This field is a string, up to 256 ASCII hex encoded characters (0x20-0x7E). |
+| `PriceDataSeries`   | Array     | Array         | Yes       | An array of up to 10 `PriceData` objects, each representing the price information for a token pair. More than five `PriceData` objects require two owner reserves. |
+| `LastUpdateTime`    | Number    | UInt32        | Yes       | The time the data was last updated, represented in the [ripple epoch time](https://xrpl.org/basic-data-types.html#specifying-time). |
+| `URI`               | String    | Blob          | No        | An optional Universal Resource Identifier to reference price data off-chain. This field is limited to 256 bytes. |
+| `AssetClass`        | String    | Blob          | Yes       | Describes the type of asset, such as "currency", "commodity", or "index". This field is a string, up to 16 ASCII hex encoded characters (0x20-0x7E). |
+| `OwnerNode`         | ???       | ???           | Yes       | ??? |
+| `PreviousTxnID`     | String    | UInt256       | Yes       | The hash of the previous transaction that modified this entry. |
+| `PreviousTxnLgrSeq` | String    | UInt32        | Yes       | The ledger index that this object was most recently modified or created in. |
+
+
+### PriceData Fields
+
+| Field               | JSON Type | Internal Type | Required? | Description |
+|---------------------|-----------|---------------|-----------|-------------|
+| `BaseAsset`         | String    | Currency      | Yes       | The primary asset in a trading pair. Any valid identifier, such as a stock symbol, bond CUSIP, or currency code is allowed. For example, in the BTC/USD pair, BTC is the base asset; in 912810RR9/BTC, 912810RR9 is the base asset. |
+| `QuoteAsset`        | String    | Currency      | Yes       | The quote asset in a trading pair. The quote asset denotes the price of one unit of the base asset. For example, in the BTC/USD pair, BTC is the base asset; in 912810RR9/BTC, 912810RR9 is the base asset. |
```

Comment:
"in the BTC/USD pari, USD is the quote aset; in 912810RR9/BTC, BTC is the quote asset."